pcluster/pcluster_stack.py

```diff
@@ -299,13 +394,16 @@ def __init__(self, scope: cdk.Construct, id: str, **kwargs) -> None:
                 'Cloud9Environment': cloud9_instance.environment_id,
                 'BootstrapPath': 's3://%s/%s' % (bootstrap_script.s3_bucket_name, bootstrap_script.s3_object_key),
                 'Config': config,
+                'BaseOSChoice': base_os_choice,
                 'VPCID': vpc.vpc_id,
                 'MasterSubnetID': vpc.public_subnets[0].subnet_id,
                 'ComputeSubnetID': vpc.private_subnets[0].subnet_id,
+                'AdditionalSG': pcluster_sg.security_group_id,
                 'PostInstallScriptS3Url':  "".join( ['s3://', pcluster_post_install_script.s3_bucket_name,  "/", pcluster_post_install_script.s3_object_key ] ),
                 'PostInstallScriptBucket': pcluster_post_install_script.s3_bucket_name,
                 'S3ReadWriteResource': data_bucket.bucket_arn,
                 'S3ReadWriteUrl': 's3://%s' % ( data_bucket.bucket_name ),
+                'FsxID': fsx_lustre_filesystem.ref,
```

Comment:
can this just be `FSxID` or is it more than that?

Reply:
Done.